### ISSUE TITLE
Categories Missing

### ISSUE THREAD

**Issue by leoplaw** [WebDevStudios/custom-post-type-ui#317]
All features and methods for accessing and assigning categories to all post types are missing.

Have deactivated all plugins, and when CPTUI is activated, category functionality disappears.
Also with TwentyFifteen theme and CPTUI activated, category functionality is gone.
Deactivating CPTUI brings category features and functionality back.

WordPress 4.2.2
CPTUI 1.1.1

**Comment by tw2113** [comment]
Could you do an export of the post type and/or taxonomy setting you're using, via the Import/Export page and provide the results to me?

I wager it's something with the settings causing issues/errors

**Comment by leoplaw** [comment]
OK, will do, but let you know, I just installed CPTUI 1.1.2 and the problem still persists.
I'll post the export here.

**Comment by tw2113** [comment]
I would have been surprised if those changes did anything, to be honest. None of them apply to similar situations ;)

**Comment by leoplaw** [comment]
// Post Types
{"artwork":{"name":"artwork","label":"Artworks","singular_label":"Artwork","description":"Paintings, Graphic Works, Sculpture","public":"1","show_ui":"1","has_archive":"0","exclude_from_search":"0","capability_type":"post","hierarchical":"0","rewrite":"1","rewrite_slug":"","rewrite_withfront":"1","query_var":"1","menu_position":"","show_in_menu":"1","show_in_menu_string":"","menu_icon":"","supports":["title","editor","excerpt","trackbacks","custom-fields","comments","revisions","thumbnail","author","page-attributes","post-formats"],"taxonomies":[],"labels":{"menu_name":"","add_new":"","add_new_item":"","edit":"","edit_item":"","new_item":"","view":"","view_item":"","search_items":"","not_found":"","not_found_in_trash":"","parent":""}},"contact":{"name":"contact","label":"Contacts","singular_label":"Contact","description":"","public":"1","show_ui":"1","has_archive":"0","exclude_from_search":"1","capability_type":"post","hierarchical":"0","rewrite":"1","rewrite_slug":"","rewrite_withfront":"1","query_var":"1","menu_position":"","show_in_menu":"1","show_in_menu_string":"","menu_icon":"","supports":["title","editor","excerpt","trackbacks","custom-fields","comments","revisions","thumbnail","author","page-attributes","post-formats"],"taxonomies":["category","post_tag"],"labels":{"menu_name":"","add_new":"","add_new_item":"","edit":"","edit_item":"","new_item":"","view":"","view_item":"","search_items":"","not_found":"","not_found_in_trash":"","parent":""}}}

// Taxonomies
{"technique":{"name":"technique","label":"Techniques","singular_label":"Technique","hierarchical":"0","show_ui":"1","query_var":"1","rewrite":"1","rewrite_slug":"","show_admin_column":"1","labels":{"search_items":"","popular_items":"","all_items":"","parent_item":"","parent_item_colon":"","edit_item":"","update_item":"","add_new_item":"","new_item_name":"","separate_items_with_commas":"","add_or_remove_items":"","choose_from_most_used":""},"object_types":["artwork"]},"material":{"name":"material","label":"Materials","singular_label":"Material","hierarchical":"0","show_ui":"1","query_var":"1","rewrite":"1","rewrite_slug":"","show_admin_column":"0","labels":{"search_items":"","popular_items":"","all_items":"","parent_item":"","parent_item_colon":"","edit_item":"","update_item":"","add_new_item":"","new_item_name":"","separate_items_with_commas":"","add_or_remove_items":"","choose_from_most_used":""},"object_types":["artwork"]},"category":{"name":"category","label":"Categories","singular_label":"category","hierarchical":"false","show_ui":"true","query_var":"true","query_var_slug":"","rewrite":"true","rewrite_slug":"","rewrite_withfront":"1","rewrite_hierarchical":"0","show_admin_column":"false","labels":{"menu_name":"","all_items":"","edit_item":"","view_item":"","update_item":"","add_new_item":"","new_item_name":"","parent_item":"","parent_item_colon":"","search_items":"","popular_items":"","separate_items_with_commas":"","add_or_remove_items":"","choose_from_most_used":"","not_found":""},"object_types":["artwork"]}}

**Comment by tw2113** [comment]
Your issue is the "category" taxonomy you're trying to register. https://cloudup.com/cX0Kr4yWnI0

WordPress already has "category" as a taxonomy out of the box and you're overwriting it, not creating a second one alongside the out-of-box one. If you're set on having a second one of similar name, I recommend, at least, changing your "category" slug to something different, and you should be good to go at that point.

**Comment by leoplaw** [comment]
So what happens if I delete that Taxonomy? Will I loose them, or are they now stored in the default WP Category. You know the funny thing is this actually worked for a long time.

**Comment by tw2113** [comment]
I have to believe the out of box taxonomy would pick them up, but I can't confirm firsthand.

Otherwise, you'd need to edit the appropriate fields in wp_term_taxonomy table to be updated to the newly renamed taxonomy slug.

**Comment by leoplaw** [comment]
Confirmed, the categories were not deleted. I also found the extra settings I was looking for in the plugin to have standard categories appear in the custom post type, and hence not have to recreate it.
May I suggest a new bit of functionality for the plugin?
When an attempt to register a new taxonomy with a slug the same as an existing one is made that it is disallowed and an error message is displayed.
